pull_request_template.md

```diff
@@ -1,19 +1,28 @@
+<!-- Comment in the sections you need for your PR! -->
```

Comment:
Since this requires removing HTML comments, I think this should be rephrased.

```suggestion
<!-- Uncomment and update the sections you need for your PR! -->
```